lib/facter/java_version.rb

```diff
@@ -12,9 +12,21 @@
 # Notes:
 #   None
 Facter.add(:java_version) do
-  setcode do
-    if Facter::Util::Resolution.which('java')
+  if Facter::Util::Resolution.which('java')
```

Comment:
Good catchk, I guess it's just a cut'n pasto error I didn't noticed before. Being busy at work today, have to check later today when back home, hope to find a bit of time then.